spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/tf/Session.scala

```diff
@@ -29,6 +29,9 @@ import org.apache.spark.SparkContext
 import org.apache.spark.api.java.JavaRDD
 import org.apache.spark.rdd.RDD
 import org.tensorflow.framework.{GraphDef, NodeDef}
+import com.google.protobuf.ByteString
```

Comment:
val sc = SparkContext.getOrCreate()

Should not init a sc in this way. Use Engine.init

Reply:
val optMethod = new SGD[T]()
    opt.setOptimMethod(optMethod).setEndWhen(endWhen)
      .optimize()

Always use a SGD, user cannot change?

Reply:
I can submit another pr to fix this problem.

Reply:
why not fix in this PR?